tests/tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+current_directory=$(pwd)
+test_framework_directory="$current_directory/jenkinsfile-runner-test-framework"
+sh_unit_directory="$test_framework_directory/shunit2"
+
+version="256.0-test"
+CWP_version="1.3"
+jenkinsfile_runner_tag="jenkins-experimental/jenkinsfile-runner-test-image"
+
+downloaded_cwp_jar="to_update"
+
+. $test_framework_directory/utilities/utils.inc
+. $test_framework_directory/utilities/cwp/custom-war-packager.inc
+. $test_framework_directory/utilities/jfr/jenkinsfile-runner.inc
+
+oneTimeSetUp() {
+  downloaded_cwp_jar=$(download_cwp "$test_framework_directory" "$CWP_version")
+  downloaded_cwp_jar=$(remove_string "$downloaded_cwp_jar" "CWP downloaded as ")
+}
+
+setUp() {
+  echo "Initializing the test case. Ignoring so far"
+}
+
+test_with_tag() {
+  jfr_tag=$(generate_docker_image_from_jar "$test_framework_directory" "$downloaded_cwp_jar" "$version" "$current_directory/test_resources/test_with_tag/packager-config.yml" "$jenkinsfile_runner_tag" | grep 'Successfully tagged')
```

Comment:
Why the grep?

Reply:
Just for performance. The execution of the `java -jar cwp.jar` displays tons of logs and a variable in shell-script has a limited (although high) number of characters to store. That way the variable only will contain the text we are interested in.

Reply:
I see. Maybe we could think in redirecting the output to some kind of file to be consulted later? Something that can be done in the future though.

Reply:
That's something to be thought once the hooks are developed because any concrete test depends on the framework's (library/hooks/helpers) design. For the moment, those tests are just a start point.